__tests__/app.test.js

```diff
@@ -62,6 +62,14 @@ describe("/api/articles/:article_id", () => {
                 expect(body.article).toHaveProperty("author", "icellusedkars");
             });
     });
+    test("GET: 200 - responds with an article by its id with a comment_count property", () => {
+        return request(app)
+            .get("/api/articles/3")
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.article).toHaveProperty("comment_count");
```

Comment:
This test doesn't tell me if the comment count is right 👀

Reply:
Thanks Jo - I've added in:

`expect(body.article.comment_count).toBe(2);`

However is there a better way to do this, as say if someone were to post a comment it would change the comment_count? Or is that I guess the whole point of the test dashboard where we re-seed so it should always pass in this case. I think I answered my own question halfway through typing haha but just wanted to confirm!